lib/src/realm_object.dart

```diff
@@ -18,13 +18,16 @@
 
 import 'dart:async';
 import 'dart:ffi';
+import 'dart:mirrors';
 
 import 'list.dart';
 import 'native/realm_core.dart';
 import 'realm_class.dart';
 
+typedef DartDynamic = dynamic;
```

Comment:
I suggest we don't hide this under our own name

Reply:
This is just a typedef - it's not going to affect users. The reason why I need to typedef it is to avoid a collision with the `dynamic` property on `RealmObject`.

Reply:
Won't users be hit by the same?